drivers/usb/udc/udc_numaker.c

```diff
@@ -171,9 +171,10 @@ static inline void numaker_usbd_sw_connect(const struct device *dev)
 	base->INTSTS = base->INTSTS;
 
 	/* Enable relevant interrupts */
-	base->INTEN = USBD_INT_BUS | USBD_INT_USB | USBD_INT_FLDET |
-		      IF_ENABLED(CONFIG_UDC_ENABLE_SOF, (USBD_INT_SOF |))
-		      USBD_INT_WAKEUP;
+	base->INTEN = USBD_INT_BUS | USBD_INT_USB | USBD_INT_FLDET | USBD_INT_WAKEUP;
+#if defined(CONFIG_UDC_ENABLE_SOF)
+	base->INTEN |= USBD_INT_SOF;
+#endif
```

Comment:
>This refines code style on CONFIG_UDC_ENABLE_SOF to be more readable.

No, it absolutely does not. Please keep it as it is.

Reply:
Reverted back this change